src/rules/map-keys-always-quoted/index.js

```diff
@@ -0,0 +1,83 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+import valueParser from "postcss-value-parser";
+
+export const ruleName = namespace("map-keys-always-quoted");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Quote function used with an already-quoted string"
```

Comment:
I forgot this when I was reviewing the previous rules, but the naming convention in stylelint error messages is `"Expected [something] [in some context]"` and `"Unexpected [something] [in some context]"`. We should try to stick to that message format in the future.

https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#determine-violation-messages